.github/workflows/android-instrumented-tests-ci.yml

```diff
@@ -0,0 +1,44 @@
+name: RadioDroid Instrumented Tests CI
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  integration-test:
+    runs-on: macOS-10.14
+    strategy:
+      matrix:
+        api: [16, 21, 27, 29]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - uses: malinskiy/action-android/install-sdk@release/0.0.6
+      - name: Instrumented tests
+        uses: malinskiy/action-android/emulator-run-cmd@release/0.0.6
+        timeout-minutes: 35
+        with:
+          cmd: ./gradlew connectedFreeDebugAndroidTest
+          api: ${{ matrix.api }}
+          tag: default
+          abi: x86
+          hardwareProfile: Nexus 6
```

Comment:
`disableAnimations: true` seems to be essential here, see https://github.com/morckx/RadioDroid/commit/27cacd884b892ee02f52dac89c2ec6a836f6cd1c